### PR TITLE
fix: should work in old version react

### PR DESCRIPTION
**Pull request by zombieJ**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      Fix Table throw `React.createRef is not a function` error  in old version of React .     |
| 🇨🇳 Chinese |      修复 Table 在旧版 React 会报 `React.createRef is not a function` 错误信息。     |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/table/Table.tsx [hunk 1]
```diff
@@ -239,11 +239,10 @@ class Table<T> extends React.Component<TableProps<T>, TableState<T>> {
 
   row: React.ComponentType<any>;
 
-  rcTable: React.RefObject<any>;
+  rcTable: any;
 
   constructor(props: TableProps<T>) {
     super(props);
-    this.rcTable = React.createRef();
```
Comment (afc163) [inline components/table/Table.tsx]: 为啥之前 react 15 的 ci 没挂。。

Comment (zombieJ) [reply components/table/Table.tsx]: 还在找原因

**Comment by zombieJ** [comment]
配置改了一下，不知道是不是这个原因。加了一个版本信息打印方便未来遇到同样问题 debug：
截屏2019-10-17下午3 33 01

**Comment by afc163** [comment]
按照 ci 的里命令一行行操作，本地可以调试 React 15 的测试用例的。

**Pull request merged**